projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -70,6 +71,9 @@ public function visitor_can_view_content( $valid_plan_ids ) {
 		}
 
 		if ( $is_valid_token ) {
+			if ( 'subscribers' === $access_level ) {
+				return 'active' === $payload['blog_sub'];
```

Comment:
constants? 

How does everyone else feel about this.  It annoys me that we have so many free strings in the code, but maybe that's my java background...